addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -342,10 +342,10 @@ def _import_invoice_ubl_cii(self, invoice, file_data, new=False):
 
         return True
 
-    def _import_retrieve_and_fill_partner(self, invoice, name, phone, mail, vat, country_code=False):
+    def _import_retrieve_and_fill_partner(self, invoice, name, phone, mail, vat, country_code=False, domain=False):
         """ Retrieve the partner, if no matching partner is found, create it (only if he has a vat and a name)
         """
-        invoice.partner_id = self.env['res.partner']._retrieve_partner(name=name, phone=phone, mail=mail, vat=vat)
+        invoice.partner_id = self.env['res.partner']._retrieve_partner(name=name, phone=phone, mail=mail, vat=vat, domain=domain)
         if not invoice.partner_id and name and vat:
             partner_vals = {'name': name, 'email': mail, 'phone': phone}
```

Comment:
we could potentially use peppol_eas & endpoint from the domain (if exists ofc) and populate them on the partner too. but feel free to ignore this

Reply:
I added a kwargs, because otherwise the signature of the function is super long and it offers more flexibility if we want to add fields in stable, what do you think ?

Reply:
it's fine for me, but I prefer if las confirms :sweat_smile:

Reply:
I don't like the kwargs here :see_no_evil: